packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1425,6 +1429,40 @@ class OffsetLayer extends ContainerLayer {
       scene.dispose();
     }
   }
+
+  /// Capture an image of the current state of this layer and its children.
+  ///
+  /// The returned [ui.Image] will be offset by the top-left corner of [bounds],
+  /// and have dimensions equal to the size of [bounds] multiplied by [pixelRatio].
+  ///
+  /// Unlike [toImage], this creates a GPU-resident texture which has a backend
```

Comment:
This paragraph sort-of leaves me as a developer wondering when I should use this over the other. Can we add some guidance?

Reply:
I think it would still be good to have some more details in here about when to use this and when to prefer toImage.

For example:

```
Prefer using this method when the image will immediately and exclusively be drawn via [Canvas.drawImage] and related [Canvas] APIs, and manipulation or transformation of the image will only come via [Canvas] and [Paint] API usage for drawing to the screen.

Prefer using [toImage] when the program needs direct access to the image bytes, for example to save them to disk or send them over a network connection, or will otherwise require direct access to a [ByteData] representation of the image.
```